services/mgmt/network/src/package_2021_03_preview/operations.rs

```diff
@@ -36130,7 +36130,12 @@ pub mod ip_groups {
 }
 pub mod network_security_perimeters {
     use crate::models::*;
-    pub async fn get(operation_config: &crate::OperationConfig) -> std::result::Result<NetworkSecurityPerimeter, get::Error> {
+    pub async fn get(
+        operation_config: &crate::OperationConfig,
+        subscription_id: &str,
+        resource_group_name: &str,
+        network_security_perimeter_name: &str,
+    ) -> std::result::Result<NetworkSecurityPerimeter, get::Error> {
```

Comment:
:thumbsup:

Prior to this change, the generated rust code wouldn't build, as `subscription_id`, `resource_group_name`, and `network_security_perimeter_name` were not defined.